src/components/Transaction/TransactionCard.vue

```diff
@@ -0,0 +1,132 @@
+<script lang="ts">
+import { defineComponent, ref, computed } from 'vue';
+import { copyToClipboard } from 'quasar';
+import { useStore } from 'src/store';
+
+export default defineComponent({
+  name: 'TransactionsTable',
+  setup() {
+    const store = useStore();
+    return {
+      transaction: computed(() => store.state.transaction.transactionId),
+      transactionData: computed(() => store.state.transaction.transaction),
+      blockNum: computed(() => store.state.transaction.blockNum),
+      timestamp: computed(() => store.state.transaction.timestamp),
+      executed: computed(() => store.state.transaction.executed),
+      irreversable: computed(() => store.state.transaction.irreversable),
+      cpuUsage: computed(() => store.state.transaction.cpuUsage),
+      netUsage: computed(() => store.state.transaction.netUsage),
+      actionsTraces: ref<string>(''),
+      actionNum: computed(() => store.state.transaction.actionCount)
+    };
+  },
+  methods: {
+    copy(value: string) {
+      copyToClipboard(value)
+        .then((): void => {
+          this.$q.notify({
+            color: 'green-4',
+            textColor: 'white',
+            message: 'Copied to clipboard',
+            timeout: 1000
+          });
+        })
+        .catch(() => {
+          this.$q.notify({
+            color: 'red-8',
+            textColor: 'white',
+            message: 'Could not copy',
+            timeout: 1000
+          });
+        });
+    },
+    numberWithCommas(x: number) {
+      if (!x) return 0;
+      return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+    },
+    formatDate(date: string): string {
+      return new Date(date).toLocaleDateString('en-US', {
+        month: 'long',
+        year: 'numeric',
+        day: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+        second: 'numeric'
+      });
+    }
+  }
+});
+</script>
+
+<template lang="pug">
+.row.full-width.justify-center
+  .col-xs-12.col-md-8.col-lg-6
+    q-card(flat class="transaction-card")
+      .q-pa-md-md.q-pa-sm-sm.q-pa-xs-xs.q-pa-xl-lg
+        q-card-section.q-pl-md
+          div(class="text-h4 text-bold") Transaction
+        
+        q-card-section.q-pt-none
+          .row.items-center
+            .col-11.text-bold.ellipsis {{transaction}}
+            .col-1
+              q-btn.float-right( @click="copy(transaction)" flat round color="black" icon="content_copy" size='sm')
+        //- q-card-section.q-pt-none
+        //-   q-btn.button-primary( @click="copyTransactionId" flat label="MSIG Template")
```

Comment:
remove unless it is a placeholder for future functionality

Reply:
Yes I guess we will have the MSIG sometime in near future so this will be used then.